src/style.scss

```diff
@@ -77,7 +77,7 @@
 
 @media (pointer: fine) {
   body {
-    overscroll-behavior-y: none;
+    overflow: overlay;
```

Comment:
Why did you remove `overscroll-behavior-y`? Also `overflow: overlay` is [deprecated](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow#values) and won't work in Firefox.